PythonSetupForIFTPipeline.jl/setup.jl

```diff
@@ -1,15 +1,20 @@
-using Pkg
-using Conda
-
 # Instantiate all the dependencies
+using Pkg
 Pkg.instantiate()
 
 # Initialize the Conda environment
-ENV["PYTHON"]=""
-Pkg.build("PyCall")
+Pkg.build("Conda")
 
 # Add the dependencies to the Conda environment
-Conda.runconda(Conda.Cmd(["env", "update", "-n", "base", "--file", joinpath(@__DIR__, "environment.yaml")]))
+using Conda
```

Comment:
When I tried running this locally and on Oscar, it couldn't find the Conda package. After adding the line `Pkg.add("Conda")` I was able to run the script. Is there a particular version of the Julia module on oscar that is recommended where perhaps conda already would be installed?

Reply:
I hadn't gotten this to run on Oscar locally at all – only with the Docker. It's great news that you managed to get it to work.

I don't think there's a way to include a version on Oscar where it's already installed, just because of the way Julia handles packages (it reinstalls all the dependencies for just your user, in the `~/.julia` directory). What I'd expect to happen is that the line `Pkg.instantiate()` will use the `Project.toml` file to install all the dependencies. But depending on whether or not there's already a `~/.julia` directory and whether there's already a conda environment in there, it might fail. 

When I did the following in the `/oscar/data/mmart119/jholla10/ice-floe-tracker-pipeline` directory, it worked:
 ```
rm -rf ~/.julia  # remove the julia install completely – sledgehammer to crack a nut
# Manually check that I don't have a ~/.condarc file which might mess up where the conda installation happens
julia --project="PythonSetupForIFTPipeline.jl" PythonSetupForIFTPipeline.jl/setup.jl
```

Does that work for you? What error messages do you get?